packages/trader/src/Assets/Trading/Categories/trade-categories.tsx

```diff
@@ -14,353 +29,53 @@ const TradeCategories = ({ category, onClick }: { category?: string; onClick: ()
                 TradeTypeTemplate = <AccumulatorTradeDescription onClick={onClick} />;
                 break;
             case 'rise_fall':
-                TradeTypeTemplate = (
-                    <React.Fragment>
-                        <Text as='p'>
-                            {localize(
-                                'If you select "Rise", you win the payout if the exit spot is strictly higher than the entry spot.'
-                            )}
-                        </Text>
-                        <Text as='p'>
-                            {localize(
-                                'If you select "Fall", you win the payout if the exit spot is strictly lower than the entry spot.'
-                            )}
-                        </Text>
-                        <Text as='p'>
-                            {localize(
-                                'If you select "Allow equals", you win the payout if exit spot is higher than or equal to entry spot for "Rise". Similarly, you win the payout if exit spot is lower than or equal to entry spot for "Fall".'
-                            )}
-                        </Text>
-                    </React.Fragment>
-                );
-                break;
             case 'rise_fall_equal':
-                TradeTypeTemplate = (
-                    <React.Fragment>
-                        <Text as='p'>
-                            {localize(
-                                'If you select "Rise", you win the payout if the exit spot is strictly higher than the entry spot.'
-                            )}
-                        </Text>
-                        <Text as='p'>
-                            {localize(
-                                'If you select "Fall", you win the payout if the exit spot is strictly lower than the entry spot.'
-                            )}
-                        </Text>
-                        <Text as='p'>
-                            {localize(
-                                'If you select "Allow equals", you win the payout if exit spot is higher than or equal to entry spot for "Rise". Similarly, you win the payout if exit spot is lower than or equal to entry spot for "Fall".'
-                            )}
-                        </Text>
-                    </React.Fragment>
-                );
+                TradeTypeTemplate = <RiseFallTradeDescription />;
```

Comment:
`rise_fall_equal` and `rise_fall` have same text description, so wrote common case for them.